config/database.php

```diff
@@ -105,14 +105,14 @@
     */
 
     'redis' => [
-
+        'client' => 'predis',
```

Comment:
This should be moved to the env, since we aren't committing the env. It shouldn't be hardcoded to this file unless we also not committing this file.

Reply:
moved